charts/kong/values.yaml

```diff
@@ -25,6 +25,9 @@ deployment:
   #   - name: sidecar
   #     image: sidecar:latest
 
+# Namespace to deploy Kong.
+namespace: ""
```

Comment:
```suggestion
# Override namepsace for Kong chart resources. By default, the chart creates resources in the release namespace.
# This may not be desirable when using this chart as a dependency.
# namespace: "example"
```